src/lv_misc/lv_font.h

```diff
@@ -68,10 +68,10 @@ typedef struct _lv_font_struct
     /*Pointer to the font in a font pack (must have the same line height)*/
     uint8_t line_height;      /*The real line height where any text fits*/
     uint8_t base_line;        /*Base line measured from the top of the line_height*/
+    int8_t  box_x_ofs_min;    /*Smallest glyph box X offset*/
```

Comment:
This not exists now in font header, but can be added without signifiant problems. What are reasons for that?

mixY/maxY exists to quickly check framebuffer borders. What are minX/maxX for?

Reply:
Maybe I missed something but IMO minY/maxY is not required because the letters will fit into a `line_height` space. In other words, it doesn't matter where the glyph is vertical because it will be somwhere in `line_height`.

However, if the first glyph of the line has `x_ofs = -1` then first might be cropped. Similarly, if the last glyph has `x_ofs = 1` the last pixel might be cropped. 
Therefore it would be important to know minX/maxX to make the labels wider accordingly.

Reply:
1. Offsets are not the same than min_left/max_right, because right border seems to be max(width).
2. For left overflow it's more practical to check first glyph only, not all (that's impossible for Y axis, so maxY/minY exists). 
3. Right overflow depends on glyphs count, and having maxX will not help you anyhow.

I did not dived into code, but data usage looks strange at first glance.

Reply:
> Offsets are not the same than min_left/max_right, because right border seems to be max(width).

Let's imagine a label with `"AB"`text and let's say:
- `width('A') = 10`
- `width('B') = 8` 
- `bbox_x_ofs('A') = -2`
- `bbox_x_ofs('B') = 1`

The label object's width should be `2 + 10 + 8 +1 = 22`

> For left overflow it's more practical to check first glyph only, not all (that's impossible for Y axis, so maxY/minY exists).

Probably the first glyph can work but I can't say it for sure now. But let's try to ignore minX, maxX for now as we can add them with quickly if required. (I removed them from the template)

> Right overflow depends on glyphs count, and having maxX will not help you anyhow.

The text needs to fit into the label objects object's size. Or with auto text wrap the text needs to fit into the label object. So it's important to know where the last pixel is.

Reply:
> The text needs to fit into the label objects object's size. Or with auto text wrap the text needs to fit into the label object. So it's important to know where the last pixel is.

As far as i understand, that should be solved at api level:

1. For single line - clip region rules applied on pixels copy, no specific actions needed.
2. For multiple lines - separate function required to split string, because there may be additional requirements how to split words. And this require incrmental iteration glyph-by-glyph anyway.